src/game/atlas.rs

```diff
@@ -2,8 +2,8 @@
 
 use guillotiere::{AllocId, Allocation, AllocatorOptions, AtlasAllocator, Size};
 use indexmap::map::IndexMap;
-use serde::export::PhantomData;
 use std::collections::HashMap;
+use std::marker::PhantomData;
```

Comment:
Yeah the IDE likes to keep importing that, thought I caught them all, thanks.  ^.^;